docs/release-notes/8.0.36-28.md

```diff
@@ -0,0 +1,62 @@
+# Percona Server for MySQL 8.0.36-28 (2024-02-)
+
+Get started with [Quickstart Guide for Percona Server for MySQL](../quickstart-overview.md).
+
+[Percona Server for MySQL 8.0.36-28](https://www.percona.com/software/mysql-database/percona-server) includes all the features and bug fixes available in the
+[MySQL 8.0.36 [Community Edition](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-36.html) and enterprise-grade features developed by Percona.
+
+## Release highlights
+
+Improvements and bug fixes provided by Oracle for MySQL 8.0.36 and included in Percona Server for MySQL are the following:'
+
+* OpenSSL is now version 3.0.12.
+
+* Updated GnuPG build key (A8D3785C). This key is used to sign MySQL downloadable packages. The previous key has expired. For details, see [Signature checking using GNUPG].
```

Comment:
This is irrelevant to Percona products. Percona packages use completely different set of keys for package signing.
Please, remove.

Reply:
Instead, I would put
```
The hashing algorithm employed yielded poor performance when using a HASH field to check for uniqueness. (Bug #109548, Bug #34959356)
```
as a noticeable improvement.